### PR TITLE
Update Pike db module tables

### PR DESCRIPTION
**Pull request by dnewh**
This updates the Pike DB module with updated/new tables in preparation for the Pike 2.0 update. This adds some tables that are not used quite yet but are defined in the Identity RFC. This also includes the associated migrations for postgres and sqlite backends and relevant daemon/cli updates.

### CODE REVIEW

File: sdk/src/hex.rs [hunk 1]
```diff
@@ -63,20 +61,6 @@ pub fn parse_hex(hex: &str) -> Result<Vec<u8>, HexError> {
...
 }
 
-/// Serializes a byte array as a hex string
-///
-/// # Arguments
-///
-///  * `data`: the byte array to convert
-///  * `serializer`: the serializer to use
-#[cfg(feature = "pike")]
-pub fn as_hex<S>(data: &[u8], serializer: S) -> Result<S::Ok, S::Error>
```
Comment (peterschwarz) [inline sdk/src/hex.rs:73]: Looks like these should be part of the hex commit

File: daemon/src/rest_api/routes/organizations.rs [hunk 2]
```diff
@@ -22,16 +22,25 @@ use crate::rest_api::{
 
 use actix::{Handler, Message, SyncContext};
 use actix_web::{web, HttpResponse};
-use grid_sdk::pike::store::Organization;
+use grid_sdk::pike::store::{Organization, OrganizationMetadata};
 use serde::{Deserialize, Serialize};
-use serde_json::Value as JsonValue;
+// use serde_json::Value as JsonValue;
```
Comment (peterschwarz) [inline daemon/src/rest_api/routes/organizations.rs:27]: Commented code, here and below

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: sdk/src/pike/store/diesel/operations/fetch_organization.rs [hunk 3]
```diff
@@ -68,26 +107,59 @@ impl<'a> PikeStoreFetchOrganizationOperation
...
-            query = query.filter(organization::service_id.is_null());
-        }
-
-        query
-            .first::<OrganizationModel>(self.conn)
-            .map(Organization::from)
-            .map(Some)
-            .or_else(|err| if err == NotFound { Ok(None) } else { Err(err) })
-            .map_err(|err| PikeStoreError::InternalError(InternalError::from_source(Box::new(err))))
+        self.conn.immediate_transaction::<_, PikeStoreError, _>(|| {
```
Comment (peterschwarz) [inline sdk/src/pike/store/diesel/operations/fetch_organization.rs:110]: Why the use of `immediate_transaction` instead of just `transaction`?

**Review by RyanLassigBanks — APPROVED** [review]

File: sdk/src/pike/store/diesel/operations/fetch_organization.rs (hunk 3 above)
Comment (dnewh) [reply sdk/src/pike/store/diesel/operations/fetch_organization.rs]: This was just how all the operations in Grid were written. Updated now

**Review by peterschwarz — APPROVED** [review]

**Review by RyanLassigBanks — APPROVED** [review]

**Pull request merged**